app/cafe.py

```diff
@@ -0,0 +1,26 @@
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
+import datetime
```

Comment:
Ensure that the imports are grouped and ordered correctly. According to the checklist, standard library imports should be separated from local application imports by a blank line.